src/main/java/com/sap/oss/phosphor/fosstars/data/github/SecurityReviewsFromOpenSSF.java

```diff
@@ -83,7 +84,7 @@ public SecurityReviewsFromOpenSSF(GitHubDataFetcher fetcher) {
         if (!reviewDate.isPresent()) {
           continue;
         }
-        reviews.add(new SecurityReview(project, reviewDate.get()));
+        reviews.add(new SecurityReview(copy(project, GitHubProject.class), reviewDate.get()));
```

Comment:
It is enough to make one clone and use it for all `SecurityReview`.

Reply:
If I understand correctly, the loop occurs because the subject has a rating value. Do you know where this rating value comes from? Does it make sense to keep it in the copy?

Reply:
In fact, I just realized that `SecurityReview.subject()` is not used at all. How about we just drop it? Since this class was introduced in `1.4.0`, it should be not break anything.

Reply:
I considered this. Removing `subject` field. Then the `SecurityReview` class will hold only `date` field. It would not make sense to use this `SecurityReview` class at all. 

Then I wondered why this was introduced in the first place. For example consider a project with multiple artifacts. Then is there a possibility that each `subject` could have different `SecurityReviews`? 

Does it make sense?

Reply:
> If I understand correctly, the loop occurs because the subject has a rating value. Do you know where this rating value comes from? Does it make sense to keep it in the copy?

Yes I found it, it happens here https://github.com/SAP/fosstars-rating-core/blob/master/src/main/java/com/sap/oss/phosphor/fosstars/tool/SingleRatingCalculator.java#L127 -> The `RatingValue` is set.

Reply:
> Then the `SecurityReview` class will hold only `date` field. It would not make sense to use this `SecurityReview` class at all.

Currently, the security review scoring function doesn't use the subject. The function just checks whether or not the project had security review in the past. So, I think it still makes sense to use the class. Furthermore, #586 is going to introduce a new filed in the security review, and the scoring function is going to use the review date as well.

> Then I wondered why this was introduced in the first place. For example consider a project with multiple artifacts. Then is there a possibility that each `subject` could have different `SecurityReviews`?

When a rating is being calculated, each feature value is connected to a subject. Most of values are connected to the subject implicitly by storing them in a value set. Currently, only `SecurityReview` has an implicit link. If you drop the field `subject`, the explicit link via the value set will be there. If you'd like to calculate ratings for multiple subject, you need to put them in separate values sets anyway. Moreover, we don't really have security reviews for artifacts. I am not sure it makes sense at all.

Reply:
Ok, then I shall remove the subject and also remove the cloning part.